packages/core/botpress/src/dialog/engine.js

```diff
@@ -673,10 +675,17 @@ bp.dialogEngine.onBeforeSessionTimeout((ctx, next) => {
         try {
           args = JSON.parse(argsStr)
           args = _.mapValues(args, value => {
-            if (_.isString(value) && value.startsWith('{{') && value.endsWith('}}')) {
-              const key = value.substr(2, value.length - 4)
-              return _.get({ state: userState, s: userState, event: event, e: event }, key)
+            if (_.isString(value) && hasVariable(value)) {
+              const view = {
```

Comment:
I've touched this in a different PR (unmerged yet), can you please refactor it like this: add a line `const actionCtx = { state: userState, s: userState, event: event, e: event }` just below `args = JSON.parse(argsStr)` and then use this `actionCtx` instead of `view`

Reply:
ok